### DISCUSSION TITLE
presta:sitemaps:dump All Pages?

### DISCUSSION THREAD

**Discussion by esisl** [pimcore/pimcore#10172] [Q&A / Support]
Create sitemap console presta:sitemap:dump --base-url=http://...
But it only crawls the top-level pages!
Is it possible to force the prest to bypass all the pages?

**Comment by dvesh3** [comment]
It generates all the pages defined in Pimcore document tree in `sitemap.*.xml` and if you have dynamic pages then please implement custom generator. please see details here https://pimcore.com/docs/pimcore/current/Development_Documentation/Tools_and_Features/Sitemaps.html
